saleor/graphql/order/mutations/orders.py

```diff
@@ -210,10 +210,17 @@ def save(cls, info, instance, cleaned_input):
             user = User.objects.filter(email=instance.user_email).first()
             instance.user = user
         instance.save()
+
+        invalid_price_fields = ["shipping_address", "billing_address"]
```

Comment:
Looks like, this is OrderUpdate mutation. We need to make sure, that we invalidate prices only for orders with status DRAFT and UNCONFIRMED

Reply:
There is a check for order status inside `invalidate_order_prices` function